pennylane_qulacs/qulacs_device.py

```diff
@@ -200,10 +202,6 @@ def _apply_qubit_state_vector(self, op):
         wires = self.map_wires(op.wires)
         input_state = op.parameters[0]
 
-        if len(input_state) != 2 ** len(wires):
```

Comment:
Why were those checks removed?

Reply:
Check the PR description, they were recently added in `QubitStateVector.__init__`